src/control-plane/backend/lambda/api/test/api/quicksight-visual-utils.test.ts

```diff
@@ -0,0 +1,522 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import { applyChangeToDashboard } from '../../common/quicksight-visual-utils';
+
+describe('SQL Builder test', () => {
```

Comment:
might rename it as 'QuickSight visual management'

Reply:
fixed